style-sample.css

```diff
@@ -0,0 +1,161 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app-sample {    
+    height: 896px;
+    width: 1440px;
+    left: 0px;
+    top: 1680px;    
+    background-color: #D5FAFC;    
+}
+
+.section-heading {
+    height: 168px;
+    width: 445px;
+    top: 120px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    position: absolute;
+}
+
+.section-quotes {
+    height: 120px;
+    width: 143px; 
+    margin-top: 64px;
+    margin-left: 165px;
+}
```

Comment:
Same with other sections. Use flexbox and remove positioning. And when width and height is applied then make sure to use some rounded values like 0's and 5's. Using values like 143px or 128px looks not promising.